src/runtime/virtcontainers/qemu_arch_base.go

```diff
@@ -467,7 +481,8 @@ func (q *qemuArchBase) appendSCSIController(_ context.Context, devices []govmmQe
 
 // appendBridges appends to devices the given bridges
 func (q *qemuArchBase) appendBridges(devices []govmmQemu.Device) []govmmQemu.Device {
```

Comment:
I guess here will break the setup swap device, because the swap device will add to bridges, but there are no bridges created.

Reply:
related issue: https://github.com/kata-containers/kata-containers/issues/5627